cli/azd/pkg/apphost/generate.go

```diff
@@ -274,6 +286,60 @@ func newInfraGenerator() *infraGenerator {
 	}
 }
 
+func evaluateForOutputs(value string) (map[string]genOutputParameter, error) {
```

Comment:
A few unit tests on this function would be nice.  I think at some point we might want to rewrite this function to not use regular expressions and having coverage then would be nice.

Reply:
It is a regex b/c there could be `0, 1 or n` outputs referenced in the `string`.

Happy to add tests